### PR TITLE
Fix using enigma plugin

### LINKED ISSUE #255: enigma plugin not supported by container
Container is missing gpg binary

`PHP Error: Enigma plugin: GPG binary not found. If you are sure the GPG binary is installed, please specify the location of the GPG binary using the 'binary' driver option. in /var/www/html/plugins/enigma/lib/enigma_engine.php on line 91`

with alpine, this can be easily fixed by installing needed binary 
`"apk add gpg"`

also there could be default directory defined or environment option for providing enigma_pgp_homedir
eg:  $config['enigma_pgp_homedir'] = RCUBE_INSTALL_PATH . '/pgp';

### PR DESCRIPTION
**Pull request by pabzm**
This ensures that the "enigma" plugin is usable also on our alpine images, and without the need for manual configuration.

Fixes #255

### CODE REVIEW

**Comment by pabzm** [comment]
@thomascube I'd be happy if you could have a look. If you don't I'll merge this myself in ~2 weeks.

File: templates/docker-entrypoint.sh [hunk 1]
```diff
@@ -150,6 +150,10 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
     echo "\$config['spellcheck_uri'] = '${ROUNDCUBEMAIL_SPELLCHECK_URI}';" >> config/config.docker.inc.php
   fi
 
+  if [[ "$ROUNDCUBEMAIL_PLUGINS" = *enigma* ]]; then
+    echo "$config['enigma_pgp_homedir'] = '/var/roundcube/enigma';" >> config/config.docker.inc.php
```
Comment (thomascube) [inline templates/docker-entrypoint.sh:154]: Maybe add a line in the Readme about this directory (section "Persistent data")?

Comment (pabzm) [reply templates/docker-entrypoint.sh:154]: Very good point, thank you!

Comment (williamdes) [inline templates/docker-entrypoint.sh:154]: Maybe check the setting is not already in the file?

Comment (pabzm) [reply templates/docker-entrypoint.sh:154]: Do you mean `plugins/enigma/config.inc.php`? Or which file?

Comment (williamdes) [reply templates/docker-entrypoint.sh:154]: I did mean `config/config.docker.inc.php` but maybe the script creates it each time

**Comment by pabzm** [comment]
/remind me to merge in one week

**Review by williamdes — APPROVED** [review]

**Review by thomascube — APPROVED** [review]

**Comment by pabzm** [comment]
Thank you all for the reviews and comments!

**Pull request merged**